### DISCUSSION TITLE
Unable to solve environment when installing moose

### DISCUSSION THREAD

**Discussion by RuSmith96** [idaholab/moose#16566] [Q&A Getting Started] [answered]
Hi All, 
I am installing Moose using the Conda Moose Environment and running into difficulties when trying to install moose ->

(base) rsmith@ubuntu:~$ export PATH=$HOME/miniconda3/bin:$PATH
(base) rsmith@ubuntu:~$ conda config --add channels conda-forge
Warning: 'conda-forge' already in 'channels' list, moving to the top
(base) rsmith@ubuntu:~$ conda config --add channels idaholab
Warning: 'idaholab' already in 'channels' list, moving to the top
(base) rsmith@ubuntu:~$ conda create --name moose moose-libmesh moose-tools
Collecting package metadata (current_repodata.json): done
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: - Killed

I am unable to solve the environment and it kills the process. Any fixes/advice/recommendations?

Cheers

**Comment by loganharbour** [comment, marked as answer]
Having conda just die like that could happen when it runs out of memory. How much memory do you have on your machine?

**Reply by RuSmith96** [reply]
Hi Logan, 
My computer has 8gb but I am giving my virtual machine 2gb. Is that enough? 
Storage wise, it has 20gb. 
Cheers

**Reply by loganharbour** [reply]
I would try upping it to 3 or 4GB to see if you still get the same issue.

**Reply by RuSmith96** [reply]
Thanks Logan that has worked! Nothing like a bit of extra juice for the system!

**Reply by loganharbour** [reply]
Sure thing! Sorry that the error isn't particularly useful :-)

**Reply by maqsiddiqui** [reply]
@loganharbour Im having similar issues while updating MOOSE, can you please clarify how to increase the memory as you suggested ? Want to give it a try. Thanks.

**Reply by milljm** [reply]
@maqsiddiqui, whatever virtual machine software you are using, you'll want to search the internet on how to do that. As that is what anyone in our group will be doing (WSL, Parallels, VMware, Docker, VirtualBox, etc).
